libs/novui/postcss.config.cjs

```diff
@@ -1,7 +1,20 @@
 module.exports = {
   /**
-   * must use this syntax for vite + panda
+   * Must use array + require syntax for vite + postcss & panda
+   * https://vitejs.dev/config/shared-options.html#css-postcss
    * https://panda-css.com/docs/installation/storybook#install-panda
    */
-  plugins: [require('@pandacss/dev/postcss')()],
+  plugins: [
+    require('@pandacss/dev/postcss')(),
+    require('postcss-preset-mantine')(),
+    require('postcss-simple-vars')({
+      variables: {
```

Comment:
@antonjoel82 can you please elaborate why these variables are not part of Panda and are set separately on mantine?

Reply:
These were part of the [Mantine setup](https://mantine.dev/getting-started/#:~:text=Create%20postcss.config.cjs%20file%20at%20the%20root%20of%20your%20application%20with%20the%20following%20content%3A). Given that we haven't defined our own in our Design System and to avoid any slowdowns in the event that these are somehow used in the internals of v7, I elected to keep them. We can always remove them down the line.

Reply:
Sounds reasonable. Thanks for explaining!